athletics.lic

```diff
@@ -121,14 +222,13 @@ class Athletics
         end
         return if remaining_attempts.zero?
       end
-      Flags.add('ct-climbing-finished', 'You finish practicing your climbing')
+      Flags.add('ct-climbing-finished', 'You finish practicing your climbing', 'The rope\'s will quickly fades away')
```

Comment:
Should these flags have the `ct` prefix? That usually indicates they are used in `crossing-training`.